pkgs/development/python-modules/pathvalidate/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi
+, pytest, heapdict, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "pathvalidate";
+  version = "2.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-PJvZTH7CPpz7IR/741audfl51sCZosdF7pSQ9STzJGg=";
+  };
+
+  disabled = pythonOlder "3.5";
+
+  # Tests require pytest-discord and pytest-md-report.
```

Comment:
Can we package them instead of not testing this package?

Reply:
`pytest-md-report` is actually depends on `pathvalidate` itself, making a cycle. They also have some dependencies missing. I'm not sure they are worth packaging for a test.